content/ch-quantum-hardware/Jaynes-Cummings-model.ipynb

```diff
@@ -66,36 +66,8 @@
     "from qiskit.compiler import transpile, assemble\n",
```

Comment:
<p>Seems like this experiment doesn't work well. The amp=0.3 might be too strong for this device. Probably you can use backend calibrated measurement pulse parameters as a starting point. I have no idea to roughly estimate the amplitude beyond the critical photon limit for the following expeirment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>